v03_pipeline/lib/tasks/write_imported_callset.py

```diff
@@ -146,64 +100,19 @@ def create_table(self) -> hl.MatrixTable:
         mt = select_relevant_fields(
             mt,
             self.dataset_type,
-            self.additional_row_fields(mt),
-        )
-        # This validation isn't override-able.  If a field is the wrong
-        # type, the pipeline will likely hard-fail downstream.
-        validate_imported_field_types(
-            mt,
-            self.dataset_type,
-            self.additional_row_fields(mt),
+            additional_row_fields(
```

Comment:
Can you delete `additional_row_fields` in this file since you moved it?

Reply:
yes!  that snuck through good catch!